Tests/BCCryptoTestsRelease.xcconfig

```diff
@@ -1,2 +1,4 @@
 #include "../BCFoundation/Configs/BCTestsRelease.xcconfig"
 #include "BCCryptoTests.xcconfig"
+
+LLVM_LTO = NO
```

Comment:
What's this all about? Why is LTO turned off for the tests?

On one hand turning LTO off the for tests might not be a bad idea, but if so it would make sense to do it across the board for all projects.

On the other hand I'd hesitate, because one of the reasons I like the tests to run in release on Jenkins is to try to get the code to be as close to the final public Sketch code as possible, to catch nasty issues (eg ones introduced by LTO).